micro-rdk/src/esp32/entry.rs

```diff
@@ -98,6 +98,8 @@ pub async fn serve_web_inner(
         let (cfg_response, cfg_received_datetime) =
             client.get_config(network.get_ip()).await.unwrap();
 
+        let robot_start_time = Instant::now();
```

Comment:
My plan was to do so when your change to move DataManager to Robot goes in, I wanted to keep it separate for now because I thought it might minimize merge conflicts